components/admin_console/admin_definition.jsx

```diff
@@ -1273,6 +1273,20 @@ export default {
                                 needsUtils.stateValueFalse('LdapSettings.EnableSync'),
                             ),
                         },
+                        {
+                            type: Constants.SettingsTypes.TYPE_TEXT,
+                            key: 'LdapSettings.GroupFilter',
+                            label: t('admin.ldap.groupFilterTitle'),
+                            label_default: 'Group Filter:',
+                            help_text: t('admin.ldap.groupFilterFilterDesc'),
+                            help_text_default: '(Optional) Enter an AD/LDAP Filter to use when searching for group objects.',
+                            placeholder: t('admin.ldap.groupFilterEx'),
+                            placeholder_default: 'E.g.: "(objectClass=group)"',
+                            isDisabled: needsUtils.and(
```

Comment:
I think this must be a `needsUtils.or`. It is dissabled if `Enable` or `EnableSync` are False.

Reply:
Hmmm yes I think you are right. I was assuming it would be the same as for `userFilter` but actually `userFilter` makes sense without `EnableSync` but I don't think `groupFilter` does.

Reply:
fixed